assets/src/scripts/components/hydrograph/store.js

```diff
@@ -79,6 +77,18 @@ export const Actions = {
             type: 'SET_MEDIAN_STATISTICS',
             medianStatistics
         };
+    },
+    setLegendMarkers(key) {
+        return {
+            type: 'SET_LEGEND_MARKERS',
+            key
+        };
+    },
+    selectLegendMarkers() {
```

Comment:
I wonder if this should be called something like showLegendMarkers and have the type "Show_Legend_Markers? I'm getting a little confused with the names "display_markers" and "legend_markers"? I'm not sure the name I suggested is better.

Reply:
I'm wondering if it would make more sense to remove this in favor of a selector that figures out which markers to display from the `showSeries` and `legendMarkers` attributes.

Reply:
👍

Reply:
> Can legendMarkers also be a selector?

I was thinking about having the selector calculate the markers as part of figuring out which one's to display, so there wouldn't be a need for a selector specifically for the creation of the marker objects.